Documentation/Configuration/Typo3ConfVars/DB.rst

```diff
@@ -138,6 +138,31 @@ the connection to the database:
         Name of the database/schema to connect to. Can be used with
         MySQL/MariaDB and PostgreSQL.
 
+    ..  confval:: defaultTableOptions
+        :Path: $GLOBALS['TYPO3_CONF_VARS']['DB']['Connections'][<connection_name>]['defaultTableOptions']
+        :name: typo3-conf-vars-db-connection-name-defaultTableOptions
+        :type: array
+
+        Defines the charset and collation options for tables for MySQL/MariaDB:
```

Comment:
```suggestion
        Defines the charset and collation options when new tables are created (MySQL/MariaDB only):
```

Reply:
Not sure about this exact wording, but we should IMO specify that 'defaultTableOptions' applies to CREATE TABLE statements, the charset here is not used for existing tables or the connection itself. What do you think?

Reply:
I think your addition makes it more clear